controllers/templatesync/predicate.go

```diff
@@ -1,50 +0,0 @@
-// Copyright (c) 2023 Red Hat, Inc.
-// Copyright Contributors to the Open Cluster Management project
-
-package templatesync
-
-import (
-	policiesv1 "open-cluster-management.io/governance-policy-propagator/api/v1"
-	"sigs.k8s.io/controller-runtime/pkg/event"
-	"sigs.k8s.io/controller-runtime/pkg/predicate"
-)
-
-// templatePredicates filters out changes to policies that don't need to be
-// considered by the template-sync controller.
-func templatePredicates() predicate.Funcs {
-	return predicate.Funcs{
-		UpdateFunc: func(e event.UpdateEvent) bool {
```

Comment:
Rather than delete this predicate, could you check to see if the `oldPolicy` has a `template-error` compliance event and if so, always queue up the reconcile request?

Reply:
I suppose